lumen/ai/assistant.py

```diff
@@ -405,7 +408,7 @@ async def _get_agent(self, messages: list | str):
                 step.stream(f"Assistant decided the {agent_name!r} will provide {', '.join(deps)}.")
                 self._current_agent.object = f"## **Current Agent**: {agent_name}"
                 custom_messages = messages.copy()
-                if isinstance(subagent, SQLAgent):
+                if isinstance(subagent, (TableAgent | SQLAgent)):
```

Comment:
What's the benefit over the simpler

```suggestion
                if isinstance(subagent, (TableAgent. SQLAgent)):
```

Reply:
Ruff complains about the simpler

Reply:
Weird @hoxbro is this now preferred syntax?

Reply:
```
lumen/ai/assistant.py:411:20: UP038 Use `X | Y` in `isinstance` call instead of `(X, Y)`
```

Reply:
Disable that rule. It is slower than tuple per Andrews link.